scripts/bundle-size-collect.js

```diff
@@ -0,0 +1,37 @@
+// This script collates bundle size information from
+// minified files in apps/test-bundles/dist and writes to
+// apps/test-bundles/dist/bundlesizes.json.
+// It is uploaded as an artifact by the build definition in
+// Azure Dev Ops and used to compare baseline and PR file size
+// information which gets reported by Size Auditor
+
+var fs = require('fs');
+
+var path = 'dist';
+var sizes = {};
+var outputFilePath = 'dist/bundlesizes.json';
+
+fs.readdir(path, function(err, items) {
+  for (var i = 0; i < items.length; i++) {
+    var file = path + '/' + items[i];
```

Comment:
Could use `path.join` here as well from Node lib.

Also you can get `Dirent` classes returned if you pass a flag to `readdir`/`readdirSync`, https://nodejs.org/api/fs.html#fs_class_fs_dirent. But I don't think you _need_ most of the info it returns right now.

Reply:
@KevinTCoughlin - I've been using node for so long, yet I've never known that you can do that :) TIL!

Reply:
I did `path.join` here. But read through `Dirent` and looks rich too.